src/hb_app.erl

```diff
@@ -16,7 +16,14 @@ start(_StartType, _StartArgs) ->
     hb_sup:start_link(),
     ok = dev_scheduler_registry:start(),
     _TimestampServer = ar_timestamp:start(),
+    % Start notification manager if notify_device is configured
```

Comment:
We should probably use the `~hook@1.0` `boot` event instead as it is generic? There is also a subtlety with this path that is important: This will only ever fire _once_, even if multiple HTTP servers are started in the same BEAM environment. This isn't so relevant for prod usage, but comes up a lot in the test suite.